lib/membrane_flac_plugin/parser.ex

```diff
@@ -1,4 +1,5 @@
-defmodule Membrane.Element.FLACParser.Parser do
+defmodule Membrane.FLACParser.Parser do
+  # credo:disable-for-this-file Credo.Check.Refactor.CyclomaticComplexity
```

Comment:
Could you do some quick refactor to avoid this? If that's too much work, let's open an issue

Reply:
I tried doing this but it required some knowledge about the code from me so I left it for now. I would opt for making an issue.